grpc/README.md

```diff
@@ -1,5 +1,227 @@
 # gRPC Service
 
+## gRPC Java Client 
+
+### Setup
+
+Let's see how to configure a java project to use the Stargate gRPC API.
+The generated code based on protobuf files (`query.proto` and `stargate.proto`) is shipped with the `grpc-proto` dependency.
+
+To see a guide how it compiles the Java code see: [gRPC setup project dependencies].
```

Comment:
```suggestion
To see a guide how the Java code is compiled from the proto files see: [gRPC setup project dependencies]. This is purely background information and not required for using the client discussed here.
```

When I first clicked the link I was a little confused about how it applies to the client.